src/php/iterators/php_htable_iterator.h

```diff
@@ -10,6 +10,7 @@ typedef struct ds_htable_iterator {
     uint32_t                 position;
     ds_htable_bucket_t      *bucket;
     ds_htable_t             *table;
+    zval                    *obj;
```

Comment:
I can't seem to find any references to `GC_ADDREF` in php-src (can't find a working lxr either). Is this basically just `++GC_REFCOUNT(zend_object_ptr)`?